packages/@aws-cdk/lambda-layer-awscli/layer/requirements.txt

```diff
@@ -1 +1 @@
-awscli==1.25.70
+awscli==1.25.46
```

Comment:
This diff line will go away once this PR is released https://github.com/cdklabs/awscdk-asset-awscli/pull/35